scaletest/templates/scaletest-runner/main.tf

```diff
@@ -237,12 +255,31 @@ data "coder_parameter" "load_scenarios" {
   default = jsonencode([
     "SSH Traffic",
     "Web Terminal Traffic",
+    "App Traffic",
     "Dashboard Traffic",
   ])
 }
 
+data "coder_parameter" "load_scenario_run_concurrently" {
+  order       = 24
+  name        = "Run Load Scenarios Concurrently"
+  type        = "bool"
+  default     = false
+  description = "Run all load scenarios concurrently, this setting enables the load scenario percentages so that they can be assigned a percentage of 1-100%."
+  mutable     = true
+}
+
+data "coder_parameter" "load_scenario_concurrency_staggering" {
+  order       = 25
+  name        = "Load Scenario Concurrency Staggering"
+  type        = "number"
+  default     = 3
+  description = "The number of minutes to wait between starting each load scenario when run concurrently."
+  mutable     = true
+}
```

Comment:
nit: `load_scenario_concurrency_stagger_interval_mins`?

Reply:
I'll go with `s/interval/delay/` since I feel interval might be interpreted as something that continually happens, whereas the delay is only applied once/start.